inference-engine/src/transformations/include/transformations/common_optimizations/mark_precision_sensitive_subgraphs.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
can we incorporate this transformation into `compress_float_constants` files because it is anyway used in conjunction with `CompressFloat` transformation. It will minimize a number of files with transformation.

Reply:
Agree

Reply:
Actually I dont think this is a good idea, because it was thought that this transformation can be needed for different purposes, not only for FP16 conversion